build.html

```diff
@@ -3,4 +3,150 @@
 title: Building Your Application
 layout: default
 ---
-<h1>Building Your Application</h1>
\ No newline at end of file
+<main class="ds-base ds-l-container">
+  <a class="ds-c-skip-nav" href="#content">Skip to main content</a>
+  <div class="ds-l-row ds-u-fill--primary-darker">
+    <div class="ds-l-col--12 ds-u-text-align--right ds-u-font-size--display ds-u-color--white">
+      Building Your Application
+    </div>
+    <div class="ds-l-col--12 ds-u-text-align--right ds-u-font-size--h1 ds-u-color--white">
+      The User Guide for ACO's in Production
+    </div>
+  </div>
+
+  <div class="ds-l-row ds-u-padding-top--2">
+
+    <!-- Table of Contents -->
+    <aside class="ds-u-margin-bottom--4 ds-u-md-margin-bottom--0 ds-l-md-col--4 ds-l-lg-col--3">
```

Comment:
Is it possible to have the Table of Contents move as the user scrolls through the page? The user guide will be quite large and would be easier to jump through different sections without having to scroll to the top.

Reply:
I am not sure exactly how to do that. Don't think it's too difficult but would be be some effort. Might have to pull someone from DPC to see how they did it on their site

Reply:
@epfrankle @jmcconn120 Any thoughts on this? I think it's important to have easy navigation for a longer page.

Reply:
@mtrang1263 @PatKelsh If at all possible, it would be really helpful to have the nav persist as the user scrolls down, especially on this long page.